packages/components/src/ProgressBar/ProgressBar.test.tsx

```diff
@@ -16,4 +16,11 @@ describe("with props", () => {
     );
     expect(container).toMatchSnapshot();
   });
+
+  it("accept stepped variation", () => {
+    const { container } = render(
+      <ProgressBar currentStep={2} totalSteps={3} variation={"stepped"} />,
+    );
+    expect(container).toMatchSnapshot();
```

Comment:
let's avoid writing more snapshot tests here. I'd assert the specific things that the user would see.

- there should be 3 steps
- 2 out of the 3 are filled
- 1 out of 3 is not filled